### ISSUE TITLE
Add seed data for purchased songs

### ISSUE THREAD

**Issue by poppabear8883** [indiehd/web-api#12]
Need to add dummy data to the `song_user` pivot table. This data refers to the songs a user account has purchased.

This task should included:
- Completed migration for the `song_user` pivot table
- Completed relationships on both models (a user has many purchased songs, songs have many users that purchased it)
- Completed seeder for the model

**Comment by cbj4074** [comment]
This will work for now, but longer-term, there is additional complexity here.

When a customer adds a song to his cart and purchases it during checkout, the Order is persisted to the database. The order details contain information about which songs the customer purchased and at what price. Capturing this information is necessary for sending the Order Confirmation email, displaying the customer's Order History, and determining how much the Artist is owed.

In other words, the relationship between a customer (User) and a Song exists through a series of other relationships that are more complex than they first appear.

We can flesh this out further once we have the Digital Catalog completely ironed-out.

**Comment by poppabear8883** [comment]
- The migration task is complete on this issue 

- The `Song Model` has the `users` relationship, however the `User Model` still needs the `songs` relationship.

- No seeded data for this implementation as of right now
